README.md

```diff
@@ -34,19 +34,35 @@ A summary of the currently supported features:
 
 Grab a Phabricator token at `https://<yourserver.com>/settings/panel/apitokens/`
 
-Configure:
+Configure in Linux:
 
      export PHAB_TOKEN=cli-ABC123
     --OR--
      echo "PHAB_TOKEN: cli-ABC123" > ~/.config/phabfive.yaml
 
+Configure in Windows:
+    Phabfive looks for configuration in the following sequence in Windows environment:
+      %ALLUSERSPROFILE%\phabfive\phabfive.yaml
+      %ALLUSERSPROFILE%\phabfive\phabfive.d\*.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.yaml
+      %LOCALAPPDATA%\phabfive\phabfive.d\*.yaml
+
+    Make sure there is a minimum phabfive.yaml store in one of the location
+    e.g. echo "PHAB_TOKEN: cli-ABC123" > %LOCALAPPDATA%\phabfive\phabfive.yaml
+
+    Additionally, due to connection to Phabricator server on HTTPS requires certificate verification, it is also recommended to install [pip_system_certs](https://pypi.org/project/pip-system-certs/) to ensure system store are linked to python.
+
+      pip install pip_system_certs
```

Comment:
This package would be a great addition into the setup.py file as an additional `extras_require` entry instead of only having it in the documentation

Reply:
Would adding platform specific requirements makes better sense?

e.g.

```
setup(
    ...,
    install_requires=[
        "pip_system_certs;platform_system=='Windows'",
    ],
)
```

Reply:
@D0wn10ad Yeah acctually that makes much more sense, then you don't have to add in any special markdown readme entry either as it would install automatically where really needed

Reply:
I would still want to have a mention in readme if possible as this is really something so hidden that I had to research for days in order to use my test env to test.... but code will be added like such.